### DISCUSSION TITLE
Collapse guide tab on the right

### DISCUSSION THREAD

**Discussion by MaxfieldHutton** [IoeCmcomc/MCDatapacker#14] [Ideas]
I think it would be awesome to have a Collapsable tab like the file tree but instead of your datapack it shows the default data and functions and such for everything in vanilla

so you might have something like 

│◄ Vanilla Preview
│        ◄minecraft
│         ├  ◄loot_tables
│         ├  ◄advancements
│         └  ◄recipies
│
etc
of course you could also resize it just like the file tree

**Comment by IoeCmcomc** [comment]
Thanks for the idea. Embedding and viewing vanilla data files is doable. The only problem is that I'd like to reduce the total file size when storing identical data files across multiple Minecraft versions.
